cpp/ql/src/semmle/code/cpp/ir/dataflow/DefaultTaintTracking.qll

```diff
@@ -46,21 +46,23 @@ predicate predictableOnlyFlow(string name) {
 
 private DataFlow::Node getNodeForSource(Expr source) {
   isUserInput(source, _) and
```

Comment:
This predicate is still called from `taintedWithoutGlobals`. Should it be?

Reply:
Good point. It probably shouldn't.